pkg/kubernetes/client.go

```diff
@@ -166,6 +167,11 @@ func (c *Client) GetMasterPodLogs(ctx context.Context, namespace string) (*restC
 	return c.kubeClient.CoreV1().Pods(namespace).GetLogs(podID, &coreV1.PodLogOptions{}), nil
 }
 
+// GetWorkerPodRequest is used for getting the logs from worker pod
+func (c *Client) GetWorkerPodRequest(ctx context.Context, namespace, workerPodID string) *restClient.Request {
+	return c.kubeClient.CoreV1().Pods(namespace).GetLogs(workerPodID, &coreV1.PodLogOptions{})
```

Comment:
this not going to work for all backends, Locust uses random worker suffix on name.

my suggestion is to list all workers Pods (maybe have to use labels), sort then alphabetically, and use the index order to show the users the logs.